modules/grpc-server/src/protojure/pedestal/core.clj

```diff
@@ -210,9 +210,15 @@
 (defmethod transmit-body nil
   [ch resp-body]
   (p/resolved true))
+(defmethod transmit-body String
```

Comment:
There is already a handler for String on L197

Reply:
Apologies, fixed